src/test/resources/org/openstreetmap/atlas/checks/commands/geo_11.geojson

```diff
@@ -1 +1,10 @@
+
```

Comment:
Was this intentional?

Reply:
I believe I accidentally deleted the file and re-added by unzipping the .gz. The change must have happened when I unzipped it. I will re-add it from the origin version.